tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -200,12 +201,31 @@ module.exports = {
 
       if (targetTab === 'links') {
         await this
-          .waitForElementVisible('@sidebarLinksTab')
-          .click('@sidebarLinksTab')
+          .useXpath()
+          .waitForElementVisible(sidebarLinksTabXpath)
+          .click(sidebarLinksTabXpath)
+          .useCss()
       }
 
       return this.api.page.FilesPageElement.sharingDialog()
     },
+    /**
+     * opens sidebar for given resource
+     *
+     * @param {string} resource
+     * @returns {*}
+     */
+    openSideBar: function (resource) {
+      const resourceDivXpath = util.format(
+        this.elements.resourceDivSelector.selector,
+        xpathHelper.buildXpathLiteral(resource)
+      )
+      this
+        .useXpath()
+        .click(resourceDivXpath)
+        .useCss()
+      return this.api.page.FilesPageElement.appSideBar()
+    },
```

Comment:
you should be able to use `clickRow()` instead, that should open the side bar

Reply:
done!